### DISCUSSION TITLE
Should HT/DT score that isn't 0.75x/1.5x rankable?

### DISCUSSION THREAD

**Discussion by MizuIro-H** [ppy/osu#31333] [Ideas] [answered]
As we can already calculate pp with these opitions, why don't make these score rankable?
or make more speed(like DT 1.2x) rankable is also a good idea I think
<img width="1280" alt="スクリーンショット 2024-12-29 11 33 50" src="https://github.com/user-attachments/assets/64e537c1-4a3b-4d86-bbc0-1ee6264f3699" />

**Comment by cihe13375** [comment, marked as answer]
https://github.com/ppy/osu/discussions/29636
